db/migrate/20211104030606_add_feed_channel.rb

```diff
@@ -0,0 +1,9 @@
+class AddFeedChannel < ActiveRecord::Migration[6.1]
+  def up
+    add_column :teams, :feed_channel_id, :string
```

Comment:
One minor nit I'd like to see happen before merging; can we rename this attribute? I've been trying to be consistent with what I name columns that reference an ID from slack (they're all `slack_id` and `slack_*_id`) so, instead of `feed_channel_id`, maybe `slack_feed_channel_id`?

Reply:
oh nice can do! Also feel free to tweak the wording, I also think since it will unfurl you may not need the text as a link maybe just the link directly with something like  <#{message.permalink}|message>". Curious how unfurl will work with private channels though.

Reply:
Oh that's a good point. We probably just shouldn't post sparkles from private channels into the feed, so would you make sure that `SparkleWorker` only cross-posts public sparkles?

In the meantime I should probably hide them on the web front-end too. I already hide private channel names and just replace them with ":lock: a secret place" but sparkles from private channels should probably only be visible to the recipient

Reply:
Alright added the update to the migration as well as made sure to not post if the sparkle was from a private channel. Slightly updated the messaging as well.
![image](https://user-images.githubusercontent.com/10525357/140597349-17cf0af1-bccf-4d7e-9dc7-1d8c4589a2d4.png)